app.js

```diff
@@ -11,17 +20,20 @@ app.get("/api/topics", getApiTopics);
 
 app.get("/api/articles/:article_id", getApiArticlesById);
 
-app.get("/api/articles", getApiArticles)
+app.get("/api/articles", getApiArticles);
 
 app.get("/api/articles/:article_id/comments", getApiArticleComments);
 
-app.post("/api/articles/:article_id/comments", postComment)
-
+app.post("/api/articles/:article_id/comments", postComment);
 
+app.patch("/api/articles/:article_id", updateArticleVotes);
 
 // ------------------------------------------------------------------
 
-app.all('*', (req, res) => {
+app.use(cors());
```

Comment:
I'm not 100% sure whether it will matter but it's possible that this line `app.use(cors());` may need to go above your endpoints on line 14 for example